src/target/drivers/display/8080/320x240x16.c

```diff
@@ -54,6 +62,17 @@ int lcd_detect()
         }
     }
     if (HAS_LCD_TYPE(LCDTYPE_ILI9341)) {
+#ifdef ILI9341_RESET_PIN
+        /* Reset pin for ILI9341 */
```

Comment:
Maybe we should rename this to LCD_RESET_PIN and just have a sible reset function